sys/net/gnrc/application_layer/dhcpv6/client.c

```diff
@@ -89,6 +89,44 @@ void dhcpv6_client_conf_prefix(unsigned iface, const ipv6_addr_t *pfx,
     }
 }
 
+bool dhcpv6_client_check_ia_na(unsigned iface)
+{
+    gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
+
+    return netif->ipv6.aac_mode & GNRC_NETIF_AAC_DHCP;
+}
+
+int dhcpv6_client_add_addr(unsigned iface, ipv6_addr_t *addr)
+{
+    gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
+
+    DEBUG("DHCPv6 client: ADD IP ADDRESS\n");
+
+    return gnrc_netif_ipv6_addr_add(netif, addr, 64, 0);
+}
+
+void dhcpv6_client_deprecate_addr(unsigned iface, const ipv6_addr_t *addr)
+{
+    gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
+    int i;
+    gnrc_netif_acquire(netif);
+    i = gnrc_netif_ipv6_addr_idx(netif, addr);
+    if (i >= 0) {
+        netif->ipv6.addrs_flags[i] &= ~GNRC_NETIF_IPV6_ADDRS_FLAGS_STATE_MASK;
+        netif->ipv6.addrs_flags[i] |= GNRC_NETIF_IPV6_ADDRS_FLAGS_STATE_DEPRECATED;
+    }
+    gnrc_netif_release(netif);
+}
+
+bool dhcpv6_client_remove_addr(unsigned iface, ipv6_addr_t *addr)
+{
+    gnrc_netif_t *netif = gnrc_netif_get_by_pid(iface);
+
+    gnrc_netif_ipv6_addr_remove(netif, addr);
+
+    return false;
```

Comment:
I think I wondered before, why this return value is needed. Now that it only returns `false` it seems all the more redundant ;-).

Reply:
Oh, ehem, you are right 😅 I actually thought I removed the return value. I now changed the function in 4ca04f643d70bc0c10aee1a220ae681fc944e32b to just pass over the return value from `gnrc_netif_ipv6_addr_remove`. Should another debug message be added in case the removal fails? Or is it okay to leave it as is?

Reply:
I actually would just not return anything. The return value of `gnrc_netif_ipv6_addr_remove()` is just passed from `gnrc_netapi_set()`, but IIRC that only fails if something is wrong with the input parameters (which `gnrc_netif_ipv6_addr_remove()` will assure that they won't). So either way, I don't think a debug message is necessary here.

Reply:
Okay, great :) I removed it in 5104f6dac08009e322996930c191707bc7b3cfb9